### PR TITLE
Issue #11

### LINKED ISSUE #10: Create development setup guide
Create a guide for setting up:

1. Go development environment
2. Installing dependencies
3. Running the application

### PR DESCRIPTION
**Pull request by luyzdeleon**
Closes #10 and #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
-# pocket-core
+# Pocket Core
+Official implementation of the Pocket Network Protocol.
 
-This is a WIP of the README
+# Overview
+The Pocket Core application will allow anyone to spin up a Pocket Network Full Node, with options to enable/disable functionality and modules according to each deployment. For more information on the Pocket Network Protocol you can visit [https://pokt.network](https://pokt.network).
```
Comment (andrewnguyen22) [inline README.md]: Change "Full Node" to "full node"

File: README.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+
+# How to run it
+To run the Pocket Core binary you can use the following flags alongside the `pocket-core` executable:
+
+- `--datadir <absolute path>` to configure the data directory where the blockchain information will be stored. The default value is: `/path/to/data/dir`.
```
Comment (andrewnguyen22) [inline README.md]: Currently, the default value for --datadir isn't doing anything. Wondering if this is worth mentioning or not.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+- `--clientrpc` to enable the Client RPC endpoints. The default value is `false`.
+- `--clientrpcport <port number>` the port on which the Client RPC endpoints will run. The default value is `8545`.
+- `--relayrpc` to enable the Relay RPC endpoints. The default value is `false`.
+- `--relayrpcport <port number>` the port on which the Relay RPC endpoints will run. The default value is `8546`.
+
+# How to contribute
+Pocket Core is an open source project, and as such we welcome any contribution from anyone on the internet. Please read our [Developer Setup Guide](https://github.com/pokt-network/pocket-core/wiki/Developer-Setup-Guide) for a guide on how get started.
```
Comment (andrewnguyen22) [inline README.md]: Remove redundant "for a guide"

File: README.md [hunk 4]
```diff
@@ -1,3 +1,31 @@
...
+
+Please fork, code and submit a Pull Request for the Pocket Core Team to review and merge. We ask that you please follow the guidelines below in order to submit your contributions for review:
+
+## High impact or architectural changes
+Reach us out in our [Slack](https://www.pokt.network/slack-pokt) and start a discussion with the Pocket Core Team regarding your change before you start working, communication is key for open source projects and asynchronous contributions.
```
Comment (andrewnguyen22) [inline README.md]: Change "Reach us out in our [Slack]" to "Reach out to us on [Slack]"
Change "before you start working, communication is key" to "before you start working. Communication is key"

**Review by andrewnguyen22 — CHANGES_REQUESTED** [review]
Changes for Developer Setup Guide:
1) This guide is for anyone ~~working on~~ contributing to the development and implementation of Pocket Core. This guide will evolve alongside the development of this project and will be ~~kept~~ updated if any major changes ~~that~~ affect this setup process.
2) You can always setup symlinks if you want to have control over your directory structure while developing ~~for~~ any GO project.
3)  This will be taken into consideration when performing Pull Request reviews, ~~and~~ **as** it will ~~be helpful to~~ save the Pocket Core Team review time if your code is already formatted to standard.

File: README.md (hunk 2 above)
Comment (luyzdeleon) [reply README.md]: If we're planning on changing the behaviour to mandatory input for that flag, then yeah we could delete it from here.

**Comment by luyzdeleon** [comment]
Fixed errors on Development Setup Guide as well.

**Review by andrewnguyen22 — APPROVED** [review]

**Pull request merged**